mindjunkies/live_classes/views.py

```
@@ -36,6 +36,24 @@ def get_context_data(self, **kwargs):
         context["course"] = get_object_or_404(Course, slug=self.kwargs["slug"])
         return context
 
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)